### PR TITLE
[IMP] sale_crm: adding tags on sale orders

### PR DESCRIPTION
**Pull request by remybar**
Show the tags of a sale order in the 'Other Info/Reporting'
section using the same colors than in the CRM app.

When a quotation is created from an opportunity, existing tags
are populated from the opportunity to the sale order.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sales_team/security/ir.model.access.csv [hunk 1]
```diff
@@ -2,3 +2,6 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_crm_team,crm.team,model_crm_team,base.group_user,1,0,0,0
 access_crm_team_user,crm.team.user,model_crm_team,sales_team.group_sale_salesman,1,0,0,0
 access_crm_team_manager,crm.team.manager,model_crm_team,sales_team.group_sale_manager,1,1,1,1
+access_crm_tag,crm_tag,model_crm_tag,base.group_user,1,0,0,0
```
Comment (tde-banana-odoo) [inline addons/sales_team/security/ir.model.access.csv]: While we are at it, could you make a small separate commit that resets rights on crm tag to 0, 0, 0, 0 for users ? They have no need to see them.

File: addons/sales_team/security/ir.model.access.csv [hunk 2]
```diff
@@ -2,3 +2,6 @@ id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
...
+access_crm_tag_salesman,crm_tag salesman,model_crm_tag,sales_team.group_sale_salesman,1,1,1,0
+access_crm_tag_manager,crm_tag manager,model_crm_tag,sales_team.group_sale_manager,1,1,1,1
```
Comment (dbeguin) [inline addons/sales_team/security/ir.model.access.csv]: DBE linter: missing last empty line

File: addons/sales_team/views/crm_tag_views.xml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                            <field name="color" required="True"/>
+                        </group>
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="sales_team_tag_tree" model="ir.ui.view">
+        <field name="name">sales.team.tag.tree</field>
```
Comment (dbeguin) [inline addons/sales_team/views/crm_tag_views.xml]: I know you moved this, but while we are at it, better to apply the guidelines : 
id="module_model_view_viewtype" -> id="sales_team_crm_tag_view_tree"
name="like_id_but_with_dot_instead" -> name="sales.team.crm.tag.view.tree"

File: addons/sales_team/views/crm_tag_views.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        <field name="arch" type="xml">
+            <tree string="Tags" editable="bottom">
+                <field name="name"/>
+                <field name="color" groups="base.group_no_one"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- Tags Configuration -->
+    <record id="sales_team_tag_action" model="ir.actions.act_window">
```
Comment (dbeguin) [inline addons/sales_team/views/crm_tag_views.xml]: `id="sales_team_crm_tag_action"`

File: addons/sales_team/security/ir.model.access.csv (hunk 1 above)
Comment (remybar) [reply addons/sales_team/security/ir.model.access.csv]: ok

File: addons/sales_team/security/ir.model.access.csv (hunk 2 above)
Comment (remybar) [reply addons/sales_team/security/ir.model.access.csv]: ok

**Comment by tde-banana-odoo** [comment]
@robodoo r+

**Comment by tde-banana-odoo** [comment]
@robodoo rebase-ff

**Comment by tde-banana-odoo** [comment]
@robodoo r+ rebase-ff
